modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -801,6 +872,58 @@ definition {
 		}
 	}
 
+	@description = "LPS-163654 - Verify that it's possible to identify the Parent and Child relationship through specific labels"
+	@priority = "4"
+	test CanIdentifyParentChildRelationship {
+		task ("Given an object related to itself, in a Many-to-Many relationship") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object LPS163654",
+				objectName = "CustomObjectLPS163654",
+				pluralLabelName = "Custom Objects LPS163654");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObjectLPS163654");
+		}
+
+		task ("When the user views the relationships on the Relationship tab") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object LPS163654");
+
+			ObjectAdmin.gotoRelationshipsTab();
+
+			ObjectAdmin.addObjectRelationshipViaUI(
+				label = "Relationship",
+				object = "Custom Object LPS163654",
+				type = "Many to Many");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject163654");
+
```

Comment:
Hi Bronson, good job on your task blocks! I just had a couple of questions and/or suggestions.
For this portion of the "When" is it possible to move into the "Given" section? It looks like you're setting the Many to Many relationship here, but it can go into the "Given" based off the task block description.